### PR TITLE
Create Find Places Functionality

### LINKED ISSUE #12: Create Find Places Functionality

### PR DESCRIPTION
**Pull request by dominikJakov**
Closes #12

### CODE REVIEW

**Review by MelvinPariyadan — DISMISSED** [review]

File: tests/test_views.py [hunk 1]
```diff
@@ -209,6 +209,57 @@ def test_bad_request_create_place_user_do_not_exists(self):
...
+            "label": "caffe;relax;fun",
+        }
+
+        response = self.client.post(f'{base_url}/place', data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        response = self.client.get(
+            f'{base_url}/place?latitude={response.data["latitude"]}&longitude={response.data["longitude"]}&range=10000', format='json')
+        #range = 10000 = 10km
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```
Comment (LBeghini) [inline tests/test_views.py:261]: The test should assert that were retrieved exactly three places from the response

**Review by LBeghini — CHANGES_REQUESTED** [review]

File: unreveal/views.py [hunk 2]
```diff
@@ -167,6 +167,44 @@ def post(self, request, format=None):
...
+            openapi.Parameter('latitude', openapi.IN_PATH, type=openapi.TYPE_STRING),
+            openapi.Parameter('range', openapi.IN_PATH, type=openapi.TYPE_STRING),],
+        responses={201: serializer_class(many=True), 404: ErrorResponseSerializer}
+    )
+    def get(self, request, format=None):
+        """get request to get the place of a given longitude,latitude and range(meters)"""
+        try:
+            longitude = self.request.query_params.get('longitude')
+            latitude = self.request.query_params.get('latitude')
+            range = self.request.query_params.get('range')
```
Comment (LBeghini) [inline unreveal/views.py]: `range` is a built in function in python, this variable should be renamed

**Review by LBeghini — CHANGES_REQUESTED** [review]

**Review by Keytush — DISMISSED** [review]

File: tests/test_views.py [hunk 3]
```diff
@@ -209,6 +209,58 @@ def test_bad_request_create_place_user_do_not_exists(self):
...
+        self.assertEqual(len(response.content.decode("utf-8").strip('[]').split('},{')),2)
```
Comment (LBeghini) [inline tests/test_views.py]: What does it do?

**Review by MelvinPariyadan — DISMISSED** [review]

File: tests/test_views.py (hunk 3 above)
Comment (LBeghini) [reply tests/test_views.py]: @dominikJakov ?

**Review by LBeghini — APPROVED** [review]

**Review by MelvinPariyadan — APPROVED** [review]

**Review by Keytush — APPROVED** [review]

**Pull request merged**